source/guide_directus.rst

```diff
@@ -0,0 +1,163 @@
+.. highlight:: console
+
+.. author:: j3n57h0m45 <https://github.com/j3n57h0m45>
+
+
+.. tag:: self-hosting
+.. tag:: database
+
+.. sidebar:: About
+
+  .. image:: _static/images/directus.png
+        :align: center
+
+########
+Directus
+########
+
+.. tag_list::
+
+Directus_ is a real-time API and App dashboard for managing SQL database content.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`the shell <basics-shell>`
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here in the GitHub_ repository of the project:
+
+  * https://github.com/directus/directus/blob/main/license
+
+Prerequisites
+=============
+
+Node and npm
+------------
+
+We're using :manual:`Node.js <lang-nodejs>`, its package manager :manual_anchor:`npm <lang-nodejs.html#npm> as well as ``npx``:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '16'
+ [isabell@stardust ~]$ npm --version
+ 7.20.0
+ [isabell@stardust ~]$ npx --version
```

Comment:
As a new user this command fails. I still don't know what `npx` is here. Leave it out if we have to install it via `npm`.